kitsune/customercare/zendesk.py

```diff
@@ -0,0 +1,59 @@
+from django.conf import settings
+from zenpy import Zenpy
+from zenpy.lib.api_objects import User as ZendeskUser
+
+
+class ZendeskClient(object):
+    """Client to connect to Zendesk API."""
+
+    def __init__(self, **kwargs):
+        """Initialize Zendesk API client."""
+        creds = {
+            "email": settings.ZENDESK_USER_EMAIL,
+            "token": settings.ZENDESK_API_TOKEN,
+            "subdomain": settings.ZENDESK_SUBDOMAIN,
+        }
+        self.client = Zenpy(**creds)
+
+    def _user_to_zendesk_user(self, user):
+        fxa_uid = user.profile.fxa_uid
+        id_str = user.profile.zendesk_id
+        return ZendeskUser(
+            id=int(id_str) if id_str else None,
+            verified=True,
+            email=user.email,
+            name=user.profile.display_name,
+            locale=user.profile.locale,
+            user_fields={"user_id": fxa_uid},
+            external_id=fxa_uid,
+        )
+
+    def create_user(self, user):
+        """Given a Django user, create a user in Zendesk."""
+        zendesk_user = self._user_to_zendesk_user(user)
+        # call create_or_update to avoid duplicating users FxA previously created
+        zendesk_user = self.client.users.create_or_update(zendesk_user)
+
+        user.profile.zendesk_id = str(zendesk_user.id)
+        user.profile.save(update_fields=["zendesk_id"])
+
+        return zendesk_user
+
+    def update_user(self, user):
+        """Given a Django user, update a user in Zendesk."""
+        zendesk_user = self._user_to_zendesk_user(user)
+        zendesk_user = self.client.users.update(zendesk_user)
+        # TODO: if we're updating a user's email, zendesk will add it as a secondary identity
+        # we need to call a separate api (not implemented by zenpy) to update it to primary:
+        # https://developer.zendesk.com/api-reference/ticketing/users/user_identities/#make-identity-primary
+        return zendesk_user
+
+    def create_ticket(self, user, form):
```

Comment:
`form` should not be passed to the client but the other way around.  You can call the API either for a `save` like method or event better by adding a `send` method to the form which will call the appropriate API method